src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -9,17 +9,33 @@ import {
   SkyMediaBreakpoints
 } from '../media-query';
 
+import {
+  SkyFocusableChildrenOptions
+} from './focusable-children-options';
+
 const SKY_TABBABLE_SELECTOR = [
-  'a[href]',
-  'area[href]',
+  'a[href]:not([tabindex=\'-1\'])',
```

Comment:
What if we just listed simple element names in the `SKY_TABBABLE_SELECTOR` array, and then filter out the results of the query for elements that have `tabindex="-1"` or `"disabled"? It would make it more human-readable, and I bet the initial query would be faster, too.

```
const SKY_TABBABLE_SELECTOR = [
  'a[href]',
  'area[href]',
  etc...
];

elements = elements
  .filter(element => element.tabIndex !== -1)
  .filter(element => element.disabled !== true)
```

Reply:
Good suggestion. Done!